Packs/GetServerURL/README.md

```diff
@@ -0,0 +1,2 @@
+## GetServerURL
+Get the Cortex XSOAR Server URL.
```

Comment:
lets get rid of the xsoar here
script doc should look something like the ones here https://github.com/ElazarK/content-docs/tree/master/scripts